src/routes/+page.svelte

```diff
@@ -307,6 +307,27 @@
 	</div>
 </Section>
 
+<!-- About us -->
+<Section id="about-us" class="child:!mx-8">
+	<svelte:fragment slot="title">
+		<!-- eslint-disable-next-line svelte/no-at-html-tags -->
+		{@html c(i("home.about-us.title"))}
+	</svelte:fragment>
+	<div class="pb-10 flex items-center justify-center">
+		<div class="flex min-w-full flex-col gap-4 rounded-3xl backdrop-filter backdrop-blur border border-opacity-25 border-white bg-glass p-8 md:p-16">
```

Comment:
I like this!! Unfortunately is looks way too "normal gray background" compared to all the code you needed for it, but we have a black background so hard for us to do better 😂